internal/backend/db/dbgorm/sessions_test.go

```diff
@@ -251,3 +258,165 @@ func TestAddSessionPrintLogRecord(t *testing.T) {
 
 	testLastLogRecord(t, f, 2, s.SessionID, l)
 }
+
```

Comment:
plz combine two tests for order into one with parametrized. see events test for example

Reply:
fixed